share/tito_completion.sh

```diff
@@ -61,6 +61,7 @@ __tito_report_opts='
 __tito_tag_opts='
     --accept-auto-changelog
     --auto-changelog-message=
+    --changelog-message
```

Comment:
I'm tempted to go with just --changelog despite the consistency with the option above this.